src/components/ContentSidebar/DetailsSidebar.js

```diff
@@ -71,8 +71,12 @@ const DetailsSidebar = ({
 
     return (
         <SidebarContent hasTitle={hasTitle} title={<FormattedMessage {...messages.sidebarDetailsTitle} />}>
-            {hasVersions && <SidebarVersions onVersionHistoryClick={onVersionHistoryClick} versions={versions} />}
-            {hasNotices && <SidebarNotices file={file} />}
+            {(hasVersions || hasNotices) && (
+                <SidebarSection>
```

Comment:
@priyajeet if both of these children end up being null, we still get the styles associated with the sidebar section, so an empty div with padding gets rendered. What's the best way to prevent this without moving the logic checking if we have notices or versions here?